src/packages.builds

```diff
@@ -24,6 +24,16 @@
     <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.Private.CoreFx.NETCoreApp\Microsoft.Private.CoreFx.NETCoreApp.builds">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="Microsoft.CSharp\pkg\*.pkgproj" />
```

Comment:
You will need to pass AdditionalProperties to these projects as well. I don't recall for sure if they are needed but lets add it just in case.

Reply:
added them